src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -264,6 +272,17 @@ export class AttributeDropdownWrapped extends React.PureComponent<
         });
     }
 
+    private updateSelectionByData = (selection: IAttributeElement[], items: any[]) => {
+        return selection.map(selectedItem => {
+            const foundItem = items.find(
+                (item: any) =>
+                    (selectedItem.uri && item.uri === selectedItem.uri) ||
+                    (selectedItem.title && item.title === selectedItem.title),
+            );
+            return foundItem || selectedItem;
+        });
+    };
+
     private setupDataSource(uri: string) {
         const request = this.getAttributeElements.bind(this, uri);
```

Comment:
This could use currying private const getAttributeElements = (uri: string) =>(query: any) => ...

Reply:
fixed